### ISSUE TITLE
Many groups lost old messages (or can't load the message)

### ISSUE THREAD

**Issue by trungdn0401** [telegramdesktop/tdesktop#24461]
### Steps to reproduce

Many groups lost old messages (or can't load the message). This issue happens on Telegram App on desktop (Win10, x64), Android, and Web (web.telegram.org/k). This does not happen on TelegramX on Android.

![image](https://user-images.githubusercontent.com/22168484/166097616-3d9ba985-9b84-4783-a744-6897ff0a1d2d.png)
![image](https://user-images.githubusercontent.com/22168484/166097723-80f29772-1e58-4868-bb26-b8a07e4fba3d.png)

### Expected behaviour

It's should show the old messages in that groups.

### Actual behaviour

Missing old messages on some groups

### Operating system

Windows 10 Pro 21H2

### Version of Telegram Desktop

3.7.3

### Installation source

Static binary from official website

### Logs

```shell
[2022.04.30 11:26:14] Launched version: 3007003, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2022.04.30 11:26:14] Executable dir: C:/Users/trung/AppData/Roaming/Telegram Desktop/, name: Telegram.exe
[2022.04.30 11:26:14] Initial working dir: C:/Users/trung/AppData/Roaming/Telegram Desktop/
[2022.04.30 11:26:14] Working dir: C:/Users/trung/AppData/Roaming/Telegram Desktop/
[2022.04.30 11:26:14] Command line: C:\Users\trung\AppData\Roaming\Telegram Desktop\Telegram.exe -noupdate
[2022.04.30 11:26:14] Executable path before check: C:/Users/trung/AppData/Roaming/Telegram Desktop/Telegram.exe
[2022.04.30 11:26:14] Logs started
[2022.04.30 11:26:14] Connecting local socket to Global\0768b66da56b35618cbd486e8912bf47-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2022.04.30 11:26:14] This is the only instance of Telegram, starting server and app...
[2022.04.30 11:26:14] Moved logging from 'C:/Users/trung/AppData/Roaming/Telegram Desktop/log_start0.txt' to 'C:/Users/trung/AppData/Roaming/Telegram Desktop/log.txt'!
[2022.04.30 11:26:14] Primary screen DPI: 240
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.04.30 11:26:14] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.04.30 11:26:14] Using DirectX compiler 'C:\Users\trung\AppData\Roaming\Telegram Desktop\modules\x64\d3d\d3dcompiler_47.dll'.
[2022.04.30 11:26:14] App Info: reading settings...
[2022.04.30 11:26:14] App Info: reading encrypted settings...
[2022.04.30 11:26:14] Lang Info: Loaded cached, keys: 3661
[2022.04.30 11:26:14] OpenAL Logging Level: (not set)
[2022.04.30 11:26:14] Audio Playback Devices: OpenAL Soft on Speakers (Conexant ISST Audio)
[2022.04.30 11:26:14] Audio Playback Default Device: OpenAL Soft on Speakers (Conexant ISST Audio)
[2022.04.30 11:26:14] Audio Capture Devices: OpenAL Soft on Internal Microphone (Conexant ISST Audio)
[2022.04.30 11:26:14] Audio Capture Default Device: OpenAL Soft on Internal Microphone (Conexant ISST Audio)
[2022.04.30 11:26:15] App Info: reading accounts info...
[2022.04.30 11:26:15] App Info: reading encrypted info...
[2022.04.30 11:26:15] App Info: reading map...
[2022.04.30 11:26:15] App Info: reading encrypted map...
[2022.04.30 11:26:15] App Info: reading encrypted user settings...
[2022.04.30 11:26:15] App Info: encrypted user settings read.
[2022.04.30 11:26:15] App Info: reading encrypted mtp data...
[2022.04.30 11:26:15] MTP Info: read keys, current: 5, to destroy: 0
[2022.04.30 11:26:15] Map read time: 11
[2022.04.30 11:26:15] App Info: reading encrypted mtp config...
[2022.04.30 11:26:15] App Info: reading map...
[2022.04.30 11:26:15] App Info: reading encrypted map...
[2022.04.30 11:26:15] App Info: reading encrypted user settings...
[2022.04.30 11:26:15] App Info: encrypted user settings read.
[2022.04.30 11:26:15] App Info: reading encrypted mtp data...
[2022.04.30 11:26:15] MTP Info: read keys, current: 5, to destroy: 0
[2022.04.30 11:26:15] Map read time: 20
[2022.04.30 11:26:15] App Info: reading encrypted mtp config...
[2022.04.30 11:26:15] Export Info: Destroy top bar by controller removal.
[2022.04.30 11:26:16] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2022.04.30 11:26:16] OpenGL Profile: OpenGLES.
[2022.04.30 11:26:16] OpenGL Renderer: ANGLE (Intel, Intel(R) HD Graphics 630 Direct3D11 vs_5_0 ps_5_0, D3D11-27.20.100.8854)
[2022.04.30 11:26:16] OpenGL Vendor: Google Inc. (Intel)
[2022.04.30 11:26:16] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.7 git hash: 0bb011f9e403)
[2022.04.30 11:26:16] OpenGL Extensions: GL_EXT_draw_elements_base_vertex, GL_OES_fbo_render_mipmap, GL_EXT_blend_minmax, GL_EXT_map_buffer_range, GL_EXT_draw_buffers_indexed, GL_ANGLE_instanced_arrays, GL_EXT_robustness, GL_OES_texture_float_linear, GL_ANGLE_lossy_etc_decode, GL_ANGLE_client_arrays, GL_CHROMIUM_sync_query, GL_OES_mapbuffer, GL_OES_get_program_binary, GL_OES_depth24, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_OES_draw_buffers_indexed, GL_ANGLE_multi_draw, GL_EXT_instanced_arrays, GL_ANGLE_program_cache_control, GL_EXT_read_format_bgra, GL_OVR_multiview2, GL_ANGLE_base_vertex_base_instance, GL_ANGLE_provoking_vertex, GL_OES_texture_half_float_linear, GL_EXT_sRGB, GL_EXT_texture_format_BGRA8888, GL_KHR_parallel_shader_compile, GL_ANGLE_texture_usage, GL_OES_texture_border_clamp, GL_EXT_texture_compression_bptc, GL_OES_vertex_array_object, GL_EXT_draw_buffers, GL_EXT_texture_filter_anisotropic, GL_EXT_EGL_image_external_wrap_modes, GL_OES_compressed_EAC_RG11_signed_texture, GL_OVR_multiview, GL_EXT_multisampled_render_to_texture, GL_OES_surfaceless_context, GL_ANGLE_pack_reverse_row_order, GL_OES_compressed_ETC2_RGBA8_texture, GL_NV_fence, GL_OES_compressed_ETC2_RGB8_texture, GL_OES_standard_derivatives, GL_ANGLE_framebuffer_blit, GL_ANGLE_robust_client_memory, GL_OES_EGL_image_external_essl3, GL_CHROMIUM_lose_context, GL_EXT_debug_label, GL_EXT_occlusion_query_boolean, GL_OES_texture_npot, GL_OES_texture_half_float, GL_EXT_texture_compression_s3tc_srgb, GL_NV_pixel_buffer_object, GL_OES_compressed_ETC2_sRGB8_texture, GL_EXT_blend_func_extended, GL_EXT_texture_rg, GL_OES_EGL_image, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_ANGLE_explicit_context_gles1, GL_EXT_debug_marker, GL_ANGLE_translated_shader_source, GL_CHROMIUM_bind_uniform_location, GL_EXT_texture_compression_dxt1, GL_EXT_color_buffer_float, GL_ANGLE_explicit_context, GL_ANGLE_get_serialized_context_string, GL_EXT_texture_storage, GL_EXT_discard_framebuffer, GL_OES_depth32, GL_ANGLE_memory_size, GL_ANGLE_texture_compression_dxt3, GL_EXT_disjoint_timer_query, , GL_ANGLE_depth_texture, GL_ANGLE_framebuffer_multisample, GL_OES_packed_depth_stencil, GL_EXT_frag_depth, GL_EXT_unpack_subimage, GL_OES_texture_stencil8, GL_ANGLE_copy_texture_3d, GL_ANGLE_texture_compression_dxt5, GL_CHROMIUM_copy_texture, GL_KHR_debug, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_WEBGL_video_texture, GL_EXT_color_buffer_half_float, GL_ANGLE_request_extension, GL_NV_EGL_stream_consumer_external, GL_CHROMIUM_copy_compressed_texture, GL_NV_pack_subimage, GL_ANGLE_get_tex_level_parameter, GL_OES_draw_elements_base_vertex, GL_OES_rgb8_rgba8, GL_OES_compressed_EAC_R11_signed_texture, GL_EXT_shader_texture_lod, GL_OES_element_index_uint, GL_EXT_float_blend, GL_OES_texture_float, GL_ANGLE_texture_multisample, GL_OES_EGL_image_external, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_CHROMIUM_bind_generates_resource, GL_OES_compressed_EAC_R11_unsigned_texture, GL_ANGLE_multiview_multisample, GL_EXT_texture_norm16, GL_EXT_texture_compression_rgtc
[2022.04.30 11:26:16] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_flexible_surface_compatibility, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2022.04.30 11:26:16] OpenGL: QOpenGLContext created, version: 3.3.
[2022.04.30 11:26:16] OpenGL: [TRUE] (OverlayWidget)
[2022.04.30 11:26:17] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:18] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:18] RPC Error: request 155 got fail with code 400, error USER_NOT_PARTICIPANT
[2022.04.30 11:26:18] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:18] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:18] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:18] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:19] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:20] API Warning: not loaded minimal channel applied.
[2022.04.30 11:26:49] Skipping message, because it is already in blocks!
[2022.04.30 11:29:35] Export Info: Panel Hide By Cancel.
[2022.04.30 11:29:35] Export Info: Panel Hide Requested.
[2022.04.30 11:29:36] Export Info: Panel Closed.
[2022.04.30 11:29:36] Export Info: Stop requested.
[2022.04.30 11:29:36] Export Info: Destroying.
[2022.04.30 11:29:36] Export Info: Destroy top bar by controller removal.
[2022.04.30 11:29:36] Export Info: Controller destroyed.
[2022.04.30 11:30:41] Message Info: bad message notification received (error_code 33) for msg_id = 7092249591047570128, seq_no = 906
[2022.04.30 11:30:41] Message Info: bad message notification received, msgId 7092249591047570128, error_code 33
[2022.04.30 11:31:30] Message Info: bad message notification received (error_code 33) for msg_id = 7092249814173790356, seq_no = 12
[2022.04.30 11:31:30] Message Info: bad message notification received, msgId 7092249814173790356, error_code 33
[2022.04.30 11:31:30] Message Info: bad message notification received (error_code 33) for msg_id = 7092249814173978920, seq_no = 6
[2022.04.30 11:31:30] Message Info: bad message notification received, msgId 7092249814173978920, error_code 33
[2022.04.30 11:31:30] Message Info: bad message notification received (error_code 33) for msg_id = 7092249814173860796, seq_no = 25
[2022.04.30 11:31:30] Message Info: bad message notification received, msgId 7092249814173860796, error_code 33
[2022.04.30 11:34:11] RPC Error: request 1421 got fail with code 400, error PEER_ID_INVALID
[2022.04.30 11:36:14] Skipping message, because it is already in blocks!
[2022.04.30 11:36:21] Skipping message, because it is already in blocks!
[2022.04.30 11:40:28] API Warning: not loaded minimal channel applied.
[2022.04.30 11:40:41] Skipping message, because it is already in blocks!
[2022.04.30 11:41:12] API Warning: not loaded minimal channel applied.
[2022.04.30 11:41:12] API Warning: not loaded minimal channel applied.
[2022.04.30 11:42:15] Skipping message, because it is already in blocks!
[2022.04.30 11:44:08] RPC Error: request 2045 got fail with code 400, error PEER_ID_INVALID
[2022.04.30 11:47:14] API Warning: not loaded minimal channel applied.
[2022.04.30 11:48:35] API Warning: not loaded minimal channel applied.
[2022.04.30 14:45:57] Message Info: bad message notification received (error_code 33) for msg_id = 7092299923519126132, seq_no = 2378
[2022.04.30 14:45:57] Message Info: bad message notification received, msgId 7092299923519126132, error_code 33
[2022.04.30 14:45:57] Message Info: bad message notification received (error_code 33) for msg_id = 7092299924221604196, seq_no = 1614
[2022.04.30 14:45:57] Message Info: bad message notification received, msgId 7092299924221604196, error_code 33
[2022.04.30 14:45:58] Message Info: bad message notification received (error_code 33) for msg_id = 7092299927854084528, seq_no = 48
[2022.04.30 14:45:58] Message Info: bad message notification received, msgId 7092299927854084528, error_code 33
[2022.04.30 14:45:58] RPC Error: request 2759 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:45:58] RPC Error: request 2756 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:45:58] RPC Error: request 2761 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:45:59] RPC Error: request 2734 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:45:59] RPC Error: request 2732 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:45:59] RPC Error: request 2733 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:45:59] Message Info: bad message notification received (error_code 33) for msg_id = 7092299932175892372, seq_no = 332
[2022.04.30 14:45:59] Message Info: bad message notification received (error_code 33) for msg_id = 7092299932175983440, seq_no = 14
[2022.04.30 14:45:59] Message Info: bad message notification received, msgId 7092299932175983440, error_code 33
[2022.04.30 14:46:01] RPC Error: request 3097 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:01] RPC Error: request 3095 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:01] RPC Error: request 3102 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:01] RPC Error: request 3096 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:01] Message Info: bad message notification received (error_code 33) for msg_id = 7092299940851233660, seq_no = 757
[2022.04.30 14:46:01] Message Info: bad message notification received, msgId 7092299940851233660, error_code 33
[2022.04.30 14:46:01] Message Info: bad message notification received (error_code 33) for msg_id = 7092299940987614408, seq_no = 242
[2022.04.30 14:46:01] Message Info: bad message notification received (error_code 33) for msg_id = 7092299941287733576, seq_no = 246
[2022.04.30 14:46:01] Message Info: bad message notification received, msgId 7092299941287733576, error_code 33
[2022.04.30 14:46:01] Message Info: bad message notification received (error_code 33) for msg_id = 7092299940690054132, seq_no = 22
[2022.04.30 14:46:01] Message Info: bad message notification received, msgId 7092299940690054132, error_code 33
[2022.04.30 14:46:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092299944965574240, seq_no = 286
[2022.04.30 14:46:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092299944966325004, seq_no = 50
[2022.04.30 14:46:02] Message Info: bad message notification received, msgId 7092299944966325004, error_code 33
[2022.04.30 14:46:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092299945753037684, seq_no = 302
[2022.04.30 14:46:02] Message Info: bad message notification received, msgId 7092299945753037684, error_code 33
[2022.04.30 14:46:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092299945866246792, seq_no = 48
[2022.04.30 14:46:02] Message Info: bad message notification received, msgId 7092299945866246792, error_code 33
[2022.04.30 14:46:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092299945863610988, seq_no = 18
[2022.04.30 14:46:02] Message Info: bad message notification received, msgId 7092299945863610988, error_code 33
[2022.04.30 14:46:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092299945863875996, seq_no = 8
[2022.04.30 14:46:02] Message Info: bad message notification received, msgId 7092299945863875996, error_code 33
[2022.04.30 14:46:02] RPC Error: request 3446 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:03] RPC Error: request 3553 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:03] RPC Error: request 3559 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:03] RPC Error: request 3631 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:03] RPC Error: request 3632 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:03] RPC Error: request 3630 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:06] RPC Error: request 4081 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:06] RPC Error: request 4106 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:06] RPC Error: request 4110 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:46:19] API Warning: not loaded minimal channel applied.
[2022.04.30 14:46:21] RPC Error: request 4440 got fail with code 400, error PEER_ID_INVALID
[2022.04.30 14:46:22] API Warning: not loaded minimal channel applied.
[2022.04.30 14:47:55] API Warning: not loaded minimal channel applied.
[2022.04.30 14:49:31] RPC Error: request 4808 got fail with code 400, error PEER_ID_INVALID
[2022.04.30 14:50:10] RPC Error: request 4846 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 14:54:33] API Warning: not loaded minimal channel applied.
[2022.04.30 14:55:07] API Warning: not loaded minimal channel applied.
[2022.04.30 14:55:07] API Warning: not loaded minimal channel applied.
[2022.04.30 14:59:48] API Warning: not loaded minimal channel applied.
[2022.04.30 15:00:40] Message Info: bad message notification received (error_code 33) for msg_id = 7092303716060440536, seq_no = 2292
[2022.04.30 15:00:40] Message Info: bad message notification received, msgId 7092303716060440536, error_code 33
[2022.04.30 15:02:29] Message Info: bad message notification received (error_code 33) for msg_id = 7092304184290874728, seq_no = 142
[2022.04.30 15:02:29] Message Info: bad message notification received, msgId 7092304184290874728, error_code 33
[2022.04.30 15:04:33] Skipping message, because it is already in blocks!
[2022.04.30 15:04:42] Skipping message, because it is already in blocks!
[2022.04.30 15:06:04] RPC Error: request 6095 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 15:06:04] RPC Error: request 6093 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 15:06:04] RPC Error: request 6094 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 15:06:08] API Warning: not loaded minimal channel applied.
[2022.04.30 15:06:09] API Warning: not loaded minimal channel applied.
[2022.04.30 15:07:02] RPC Error: request 6217 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 15:07:04] RPC Error: request 6228 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 15:07:18] API Warning: not loaded minimal channel applied.
[2022.04.30 15:07:39] RPC Error: request 6310 got fail with code 400, error FILE_REFERENCE_EXPIRED
[2022.04.30 15:08:49] RPC Error: request 6392 got fail with code 400, error PEER_ID_INVALID
[2022.04.30 15:08:56] API Warning: not loaded minimal channel applied.
[2022.04.30 15:08:56] API Warning: not loaded minimal channel applied.
[2022.04.30 15:08:56] API Warning: not loaded minimal channel applied.
[2022.04.30 15:09:01] RPC Error: request 6438 got fail with code 400, error PEER_ID_INVALID
[2022.04.30 15:09:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092305872131797056, seq_no = 620
[2022.04.30 15:09:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092305872322961012, seq_no = 620
[2022.04.30 15:09:02] Message Info: bad message notification received (error_code 33) for msg_id = 7092305872321258512, seq_no = 3192
[2022.04.30 15:09:02] Message Info: bad message notification received, msgId 7092305872321258512, error_code 33
[2022.04.30 15:10:49] Skipping message, because it is already in blocks!
[2022.04.30 15:10:54] Message Info: bad message notification received (error_code 33) for msg_id = 7092306353628578088, seq_no = 796
```

**Comment by trungdn0401** [comment]
It's also happening with single-person chat, not only group or channel.

**Comment by trungdn0401** [comment]
This has been fixed in v4.0 21.06.22
